doc/nrf/known_issues.rst

```diff
@@ -2267,6 +2267,15 @@ nrfxlib
 
 Crypto
 ======
+.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSDK-20688: For ChaChaPoly, incorrect tag will be produced if plaintext is empty.
+  For encryption, empty plaintext will result in incorrect tag. For decryption, correct tags will not be accepted.
+
+.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSDK-20688: For AES GCM, calling psa_aead_update_ad multiple times will result in an incorrect tag on CryptoCell.
```

Comment:
```suggestion
NCSDK-20688: For AES GCM, calling ``psa_aead_update_ad`` multiple times will result in an incorrect tag on CryptoCell
```
Or is psa_aead_update_ad a function or something we can link to using e.g. :c:func:...?

Reply:
Oki, I see rendering is not correct using :c:func:, so we will have to use `` instead.